src/sentry/web/frontend/debug/debug_incident_activity_email.py

```diff
@@ -20,6 +20,9 @@ def get(self, request):
         activity = IncidentActivity(
             incident=incident, user=user, type=IncidentActivityType.COMMENT.value, comment="hi"
         )
+        # activity = IncidentActivity(
```

Comment:
Why are we commenting this out?

Reply:
I added it to test the email when the status changed, and left it in as a comment for someone in the future. I could remove it again and leave this file unchanged

Reply:
I'd say either put it in permanently or remove the comment, I don't mind which